addons/mail/models/mail_alias.py

```diff
@@ -15,6 +16,20 @@
 dot_atom_text = re.compile(r"^%s+(\.%s+)*$" % (atext, atext))
 
 
+@dataclass(frozen=True)
+class AliasError:
```

Comment:
Ok that's cool :) short to define. Maybe we could put in into a "tools" directory, in alias_error.py (we already have a small tool file mail_validation) ?

Reply:
I have moved it there. It is indeed better to have one file per class.